src/scripts/main.js

```diff
@@ -1,3 +1,81 @@
 'use strict';
 
-// write code here
+// // write code here
+const body = document.querySelector('body');
+const url = 'https://mate-academy.github.io/'
+  + 'phone-catalogue-static/api/phones/:phoneId.json';
+const url1 = url.substring(0, url.indexOf(':phoneId'));
+
+function getFirstReceivedDetails(id) {
```

Comment:
if it's an array name it appropriately

```suggestion
function getFirstReceivedDetails(ids) {
```